app/src/main/java/com/sunasterisk/fooddaily/data/model/FoodDetail.kt

```diff
@@ -55,4 +66,37 @@ data class FoodDetail(
             }
         }
     )
+
+    fun getRequiredTime(): String {
+        val timeRequired = StringBuilder().apply {
+            append(readyMinutes)
+            append(REQUIRED_TIME_UNIT)
+        }
+        return timeRequired.toString()
+    }
+
+    fun getPriceEstimate(): String {
+        val priceEstimate = StringBuilder().apply {
+            append(price)
+            append(PRICE_ESTIMATE_UNIT)
+        }
+        return priceEstimate.toString()
+    }
```

Comment:
Trong trường hợp này, viết như này sẽ đơn giản hơn 
khi sử dụng trong vòng lặp thì cân nhắc dùng StringBuilder 
```suggestion
    fun getPriceEstimate(): String = readMinutes + REQUIRED_TIME_UNIT
```